docs_2.0/manual_EN/3.quick-start/4.nebua-graph-crud.md

```diff
@@ -0,0 +1,474 @@
+    | -------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+    | partition_num  | Specifies the number of partitions in each replica. The suggested number is the number of hard disks in the cluster times 5. For example, if you have 3 hard disks in the cluster, we recommend that you set 15 partitions.           |
+    | replica_factor | Specifies the number of replicas in the Nebula Graph cluster. The suggested number is 3 in a production environment and 1 in a test environment. The replica number must always be an odd number for the need of quorum-based voting. |
+
+* Check for graph spaces:
+
+    ```nGQL
+    SHOW SPACES
+    ```
+
+* Use a graph space:
+
+    ```nGQL
+    USE <graph space name>
+    ```
+
+### Examples
+
+1. Use the following statement to create a graph space named `nba`.
+
+    ```nGQL
+    (root@nebula) [(none)]> CREATE SPACE nba(partition_num=15, replica_factor=1);
+    Execution succeeded (time spent 2817/3280 us)
+    ```
+
+2. Check the partition distribution with `SHOW HOSTS` to make sure the partitions are distributed in a balanced way.
+
+    ```nGQL
+    (root@nebula) [(none)]> SHOW HOSTS;
+    +-----------+-------+--------+--------------+---------------------+------------------------+
+    | Host      | Port  | Status | Leader count | Leader distribution | Partition distribution |
+    +-----------+-------+--------+--------------+---------------------+------------------------+
+    | storaged0 | 44500 | ONLINE | 1            | nba:5               | nba:5                  |
+    +-----------+-------+--------+--------------+---------------------+------------------------+
+    | storaged1 | 44500 | ONLINE | 1            | nba:5               | nba:5                  |
+    +-----------+-------+--------+--------------+---------------------+------------------------+
+    | storaged2 | 44500 | ONLINE | 1            | nba:5               | nba:5                  |
+    +-----------+-------+--------+--------------+---------------------+------------------------+
+    Got 3 rows (time spent 744/1123 us)
+    ```
+
+    If the **Leader distribution** is uneven, use `BALANCE LEADER` to redistribute the partitions. For more information, see [BALANCE](TODO) [TODO].
+
+3. Use the `nba` graph space.
+
+    ```nGQL
+    (root@nebula) [(none)]> USE nba;
+    Execution succeeded (time spent 1322/2206 us)
+    ```
+
+    You can use `SHOW SPACES` to check the graph space you created.
+
+    ```nGQL
+    (root@nebula) [nba]> SHOW SPACES;
+    +------+
+    | Name |
+    +------+
+    | nba  |
+    +------+
+    Got 1 rows (time spent 1235/1934 us)
+    ```
+
+## Create tags and edge types
+
+### nGQL syntax
+
+```nGQL
+CREATE {TAG | EDGE} {<tag name> | <edge type>}(<property name 1> <data type>
```

Comment:
```
CREATE {TAG | EDGE} {<tag_name> | <edge_type>} (<property name> <data type>
[, <property name> <data type> ...]);
```

Reply:
not necessary to write 1 2 3.

Reply:
Adding an order may help the readers better understand the matchup.